lib/web_ui/lib/src/engine/canvaskit/surface.dart

```diff
@@ -26,14 +26,14 @@ typedef SubmitCallback = bool Function(SurfaceFrame, CkCanvas);
 
 /// A frame which contains a canvas to be drawn into.
 class SurfaceFrame {
-  final CkSurface skiaSurface;
-  final SubmitCallback submitCallback;
-  bool _submitted;
-
   SurfaceFrame(this.skiaSurface, this.submitCallback)
       : _submitted = false,
         assert(skiaSurface != null), // ignore: unnecessary_null_comparison
-        assert(submitCallback != null); // ignore: unnecessary_null_comparison
+        assert(submitCallback != null);
```

Comment:
Looks like `// ignore: unnecessary_null_comparison` is still needed here?

Reply:
Removed, see other comment.